refinance/__main__.py

```diff
@@ -0,0 +1,20 @@
+from os import environ
+
+from uvicorn import run
+
+from refinance.log import configure_logging
+
+
+def main() -> None:
+    configure_logging()
+    run(
+        "refinance.app:app_factory",
+        host=environ.get("HOST", "0.0.0.0"),
```

Comment:
Is there any specific reason to set these parameters from application runtime?

Reply:
Yes. The port and host must be specified in the rantime so that the user can change it, it may be busy for example.

Reply:
I doubt that these details will be changed often, we can probably go with hardcoded values for now.